ecosystem/model.py

```diff
@@ -22,7 +22,7 @@ def __init__(self,  height, width, b, m):
         self.height = height
         self.width = width
         self.num_agents = self.width*self.height
-        self.schedule = SimultaneousActivation(self)
+        self.schedule = RandomActivation(self)
```

Comment:
I suspect there is not much difference between `RandomActivation` and `SimulatneousActivation` routines, but haven't we agreed on the latter? I might be behind on this, mind you